index.js

```diff
@@ -19,7 +19,7 @@ async function fastifyRoutesStats (fastify, opts) {
         const [
           method,
           route
-        ] = e.name.split(':')[1].split('|')
+        ] = e.name.split(':', 2)[1].split('|')
```

Comment:
Counter offer? I don’t even know at this point 🤣

I mean it gets destructed to 2 vars so better to be sure right?

```suggestion
        ] = e.name.split(':', 2)[1].split('|', 2)
```